crates/cxx-qt-build/src/lib.rs

```diff
@@ -11,6 +11,8 @@
 //! for CXX-Qt or CXX macros and generate any resulting C++ code. It also builds
 //! the C++ code into a binary with any cxx-qt-lib code and Qt linked.
 
+extern crate version_check as rustc;
```

Comment:
I find this aliasing of the crate confusing. It makes it seem like the `version_check` crate is something shipped with rustc, but it's a normal dependency from crates.io.

Reply:
Oops yup, i just copied it from their readme. Changed it to `version_check`.